cmd/jujud/agent/model/manifolds_test.go

```diff
@@ -361,7 +359,6 @@ var expectedCAASModelManifoldsWithDependencies = map[string][]string{
 		"environ-upgrade-gate",
 		"environ-upgraded-flag",
 		"not-alive-flag",
-		"valid-credential-flag",
```

Comment:
It's strange that you are removing ifCredentialValid wrap from CAAS dependencies for undertaker but not IAAS? Unles it's just a test that you've missed... Because I can still see it as expected on line 613.

Reply:
This is a transitive dependency from environ tracker - the IAAS undertaker depends on environ tracker and that's guarded by `ifCredentialValid`.